crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -112,13 +118,13 @@ pub struct BoxedPolyline2d {
 }
 impl Primitive2d for BoxedPolyline2d {}
 
-/// A triangle primitive
+/// A triangle in 2d space
```

Comment:
Nitpick, elsewhere it's capitalized
```suggestion
/// A triangle in 2D space
```